### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by mauricioabreu**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,5 +13,7 @@ services:
       - .:/code
     ports:
       - "8080:8080"
+    links:
+      - db
     depends_on:
```
Comment (csarcom) [inline docker-compose.yml]: With depends_on, app will start after db?

Comment (mauricioabreu) [inline docker-compose.yml]: Yes! Thats is what I want.

Comment (csarcom) [inline docker-compose.yml]: Is that a new feature on docker? When I was writing Qpaca I dont remeber that conf option...

Comment (mauricioabreu) [inline docker-compose.yml]: For docker-compose, yes, I assume it is "new". A couple of months ago they launched this feature. :P

**Pull request merged**